lib/utils/apiRequests.ts

```diff
@@ -0,0 +1,34 @@
+import { CustomerDTO, EmployeeDTO, ProjectDTO, TaskDTO, TimesheetDTO } from '../types/dto';
+import { get, post, put, ResponseWithStatus } from './fetch';
+
+const projectEndpointURL = `${process.env.NEXT_PUBLIC_API_URL}/project`;
```

Comment:
URLS must be set in a centralized way and never manipulated manually afterwards. Would it be a good idea if these came from the openapi docs directly?